browser/profiles/brave_renderer_updater.h

```diff
@@ -32,6 +32,10 @@ class BraveRendererUpdater : public KeyedService {
   // Initialize a newly-started renderer process.
   void InitializeRenderer(content::RenderProcessHost* render_process_host);
 
+  // TODO(cypt4): Remove this method after adding subscribtion on
+  // ExtensionRegistry and KeyringService
+  void UpdateAllRenderersForTesting();
```

Comment:
This is completely unnecessary https://github.com/brave/brave-core/pull/18172/files#r1175530352